client/src/components/FieldHelper.js

```diff
@@ -79,20 +76,18 @@ const Field = ({
       ) : (
         <InputGroup>
           {widgetElem}
-          {extraAddon && <InputGroup.Addon>{extraAddon}</InputGroup.Addon>}
           <FieldAddon id={id} addon={addon} />
         </InputGroup>
       ),
-    [addon, extraAddon, id, widgetElem]
+    [addon, id, widgetElem]
   )
   const validationState = getFormGroupValidationState(field, form)
   if (label === undefined) {
     label = utils.sentenceCase(field.name) // name is a required prop of field
   }
 
-  // setting label or extraColElem explicitly to null will completely remove these columns!
-  const widgetWidth =
-    12 - (label === null ? 0 : 2) - (extraColElem === null ? 0 : 3)
+  // setting label explicitly to null will completely remove this column!
+  const widgetWidth = 12 - (label === null ? 0 : 2)
```

Comment:
Alternatively, changing this line to:
```
+  const widgetWidth = 9 - (label === null ? 0 : 2)
```
would keep the form exactly the same width as before (if you feel it will become too wide otherwise). Opinions welcome.

Reply:
Changed to use the same width as before.